### ISSUE TITLE
VirtualBox dkms makefile fails compilation on aura, but not on pacman

### ISSUE THREAD

**Issue by Nicholas-Baron** [fosskers/aura#584]
When I install `virtualbox` and `virtualbox-host-dkms` (`sudo aura -S ...`), aura gives this error: 
```
==> dkms install vboxhost/6.1.6_OSE -k 5.4.32-1-lts
Error! Bad return status for module build on kernel: 5.4.32-1-lts (x86_64)
Consult /var/lib/dkms/vboxhost/6.1.6_OSE/build/make.log for more information.
```

Error Log from the specified file:
```
DKMS make.log for vboxhost-6.1.4_OSE for kernel 5.4.32-1-lts (x86_64)
Tue Apr 14 17:24:54 PDT 2020
make: Entering directory '/usr/lib/modules/5.4.32-1-lts/build'
make: uname: No such file or directory
make: gcc: No such file or directory
make: make: No such file or directory
make: *** [Makefile:1691: /var/lib/dkms/vboxhost/6.1.4_OSE/build] Error 127
make: Leaving directory '/usr/lib/modules/5.4.32-1-lts/build'
```

The most insteresting part of this issue is that pacman (`sudo pacman -S ...`) is fine and compiles the modules.

**Comment by fosskers** [comment]
I've actually had these problems recently when installing `vmware-workstation`. And it seemed not to have anything to do with Aura, since it would happen when I built it myself with `makepkg`!

For reference, check out the flood of complaints here: https://aur.archlinux.org/packages/vmware-workstation/

**Comment by fosskers** [comment]
Hi there, were you able to figure anything out?

**Comment by fosskers** [comment]
Any luck?

**Comment by JackCuthbert** [comment]
I'm also getting this with any `-dkms` package via `aura`. Using `PKGBUILD` files directly with `makepkg --install` or `pacman` both work without issue.

**Comment by JackCuthbert** [comment]
This appears to fix https://github.com/fosskers/aura/issues/613 for me, but dkms packages from AUR seem to fail still.

Here's what happened when I attempted to install [xpadneo](https://aur.archlinux.org/packages/xpadneo-dkms-git/).

``` 
$ aura --version
 __ _ _  _ _ _ __ _    Pacman v5.2.1 - libalpm v12.0.1
/ _` | || | '_/ _` |   Copyright (C) 2006-2019 Pacman Development Team
\__,_|\_,_|_| \__,_|   Copyright (C) 2002-2006 Judd Vinet
AURA Version 3.1.2
```
```
$ sudo aura -Aa xpadneo-dkms-git
# .... snip some logging ...
==> dkms install hid-xpadneo/0.7 -k 5.6.15-arch1-1
Error! Bad return status for module build on kernel: 5.6.15-arch1-1 (x86_64)
Consult /var/lib/dkms/hid-xpadneo/0.7/build/make.log for more information.
==> Warning, `dkms install hid-xpadneo/0.7 -k 5.6.15-arch1-1' returned 10
```

and the make.log

```
$ cat /var/lib/dkms/hid-xpadneo/0.7/build/make.log
DKMS make.log for hid-xpadneo-0.7 for kernel 5.6.15-arch1-1 (x86_64)
Thu Jun 11 20:59:32 AEST 2020
make: Entering directory '/usr/lib/modules/5.6.15-arch1-1/build'
make: uname: No such file or directory
make: gcc: No such file or directory
make: make: No such file or directory
make: *** [Makefile:1695: /var/lib/dkms/hid-xpadneo/0.7/build/src] Error 127
make: Leaving directory '/usr/lib/modules/5.6.15-arch1-1/build'
```

**Comment by fosskers** [comment]
You know what... I bet when Aura does `sudo -u <you> makepkg` to build the package, your PATH variable isn't brought through. So `make` here can't see any of those usual tools.

**Comment by fosskers** [comment]
False alarm - the `PATH` is indeed there.

**Comment by fosskers** [comment]
Whoa. Building `xpadneo-dkms-git` myself with `makepkg`, then installing with `pacman -U`, works.
**However**, trying `aura -U` fails!

**Comment by fosskers** [comment]
`htop` output when using `aura -U`. 

![2020-06-11-093341_958x538_scrot](https://user-images.githubusercontent.com/229679/84414341-b3677e80-abc6-11ea-8aa6-359e86b25527.png)

**Comment by fosskers** [comment]
From the comments on the `vmware-workstation` AUR package:

> For people with the DKMS problem: I don't have any idea what can explain this problem. It seems that your PATH is removed somewhere during the installation. Can you try to install another package using DKMS (for example virtualbox-host-dkms) to see if the problem still exists? If yes you should ask some help on Arch forum, else tell me and I will try to investigate why this problem appears only on my package.

**Comment by fosskers** [comment]
Oh... the smoking gun!
```haskell
pacmanProc :: [String] -> ProcessConfig () () ()
pacmanProc args = setEnv [("LC_ALL", "C")] $ proc "pacman" args
```
> Set the environment variables of the child process.

**Comment by fosskers** [comment]
Yup, that fixes it.

**Comment by fosskers** [comment]
I just released Aura `3.1.3` that includes the fix.

**Comment by JackCuthbert** [comment]
Legend! Works for me :ok_hand:
